pandas/tests/arrays/categorical/test_operators.py

```diff
@@ -186,10 +185,20 @@ def test_comparison_with_unknown_scalars(self):
         tm.assert_numpy_array_equal(cat != 4,
                                     np.array([True, True, True]))
 
+    def test_comparison_of_ordered_categorical_with_missing_values(self):
```

Comment:
run thru all of the comparison ops here with the compare_operators_no_eq_ne fixture

Reply:
Sounds good!